creator-node/src/snapbackSM.js

```diff
@@ -23,6 +23,16 @@ const ModuloBase = 24
 // Delay 1 hour between production state machine jobs
 const ProductionJobDelayInMs = 3600000
 
+const SyncPriority = Object.freeze({
```

Comment:
NIT - let's describe what each of these tuples is used for in a comment above

Reply:
Will do